config/crd/patches/cainjection_in_clusterlogforwarders.yaml

```diff
@@ -0,0 +1,8 @@
+# The following patch adds a directive for certmanager to inject CA into the CRD
+# CRD conversion requires k8s 1.13 or later.
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
```

Comment:
What certs do we need?

Reply:
@vparfonov In general you can fully drop the generated patches and kustomization files for cert-manager. On OpenShift we have the cert-signing service doing all the work for us.

Reply:
@periklis You mean i can remove these files: 

- [cainjection_in_clusterlogforwarders.yaml](https://github.com/vparfonov/cluster-logging-operator/tree/best_practice/config/crd/patches/cainjection_in_clusterlogforwarders.yaml)
- [cainjection_in_clusterloggings.yaml](https://github.com/vparfonov/cluster-logging-operator/tree/best_practice/config/crd/patches/cainjection_in_clusterloggings.yaml)
- [cainjection_in_outputs.yaml](https://github.com/vparfonov/cluster-logging-operator/tree/best_practice/config/crd/patches/cainjection_in_outputs.yaml)

 and this lines: [kustomization.yaml#L18L23 ](https://github.com/vparfonov/cluster-logging-operator/blob/best_practice/config/crd/kustomization.yaml#L18L23)